### PR TITLE
feat: CLI: add expiration to claim list

### PR DESCRIPTION
**Pull request by LexLuthr**
## Related Issues

Fixes https://github.com/filecoin-project/lotus/issues/11669

## Proposed Changes

Addition column called "Expiration" equivalent to Claim.TermMax + claim.TermStart

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] If the PR affects users (e.g., new feature, bug fix, system requirements change), update the CHANGELOG.md and add details to the UNRELEASED section.
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: cli/filplus.go [hunk 1]
```diff
@@ -375,21 +375,23 @@ var filplusListClaimsCmd = &cli.Command{
 			tablewriter.Col("TermMin"),
 			tablewriter.Col("TermMax"),
 			tablewriter.Col("TermStart"),
+			tablewriter.Col("Expiration"),
 			tablewriter.Col("Sector"),
 		)
 
 		for claimId, claim := range claimsMap {
 			if ts.Height() > claim.TermMax || !cctx.IsSet("expired") {
```
Comment (rvagg) [inline cli/filplus.go:383]: I think it's this line that's the problem - `--expired` needs to check `ts.Height` against the expiration height of the claim; I don't think we need an expiration column, at least that would be anther breaking change and they can calculate it anyway

**Review by rvagg — APPROVED** [review]

**Comment by rvagg** [comment]
@nezha90 care to add a +1?

**Comment by Stebalien** [comment]
Looks correct so I'm going to merge. But an additional +1 from @nezha90 would still be nice.

**Comment by nezha90** [comment]
+1 means? @rvagg

**Comment by rvagg** [comment]
@nezha90 we just wanted you to eye this and give a tick of approval, since you were the original reporter

**Comment by nezha90** [comment]
😂thanks

**Pull request merged**